lib/ajax-datatables-rails/orm/active_record.rb

```diff
@@ -38,31 +38,28 @@ def composite_search(records)
         records
       end
 
-      def build_conditions_for(query)
+      def build_conditions_for(query, regex)
```

Comment:
Looking at the examples edited in the `spec/ajax-datatables-rails/orm/active_record_filter_records_spec.rb` file, I think this method arguments could be something like:

```ruby
def build_conditions_for(query, regex = "false")
  # method code
end
```

this way, all the examples in the spec file need not to be edited :)